named_test.go

```diff
@@ -30,15 +31,15 @@ func TestCompileQuery(t *testing.T) {
 			V: []string{"name1", "name2"},
 		},
 		{
-			Q: `SELECT "::foo" FROM a WHERE first_name=:name1 AND last_name=:name2`,
+			Q: `SELECT ":foo" FROM a WHERE first_name=:name1 AND last_name=:name2`,
```

Comment:
I assume idea here is/was that `::foo` would "escape" the `:foo`; sqltoken is smarter about this as it recognizes the quotes so you don't need to use `::foo` – which is clearly better – but this is an incompatible change that will break people's SQL if they're using it like this.

As an aside, this library is "sporadically maintained", at best, so I'm not expecting a merge or feedback from the maintainer(s) any time soon, so I wouldn't rush to fix this. I'm (probably) going to use in my own library/fork I'm working on though. Just wanted to point this out.

Also might be nice to replace those four calls to testify in your library? Don't want to start a discussion about testify vs. stdlib tests, but smaller dependency trees are always nice, and with just four calls it's not that much benefit 😅

Reply:
I called this out in my PR description.  Since this is not backwards compatible it requires a major version change.   The current behavior isn't documented, but people may still be relying on it.

If getting rid of the calls to testify would mean this gets merged, I would certainly do it.

Reply:
Oh sorry, I missed that 😅

> If getting rid of the calls to testify would mean this gets merged, I would certainly do it.

I have no say in the matter at all; just something I noticed.